Tests/OpenAPIRuntimeTests/URICoder/Parsing/Test_URIParser.swift

```diff
@@ -73,7 +77,8 @@ final class Test_URIParser: Test_Runtime {
                     simpleExplode: .custom("red,green,blue", value: ["": ["red", "green", "blue"]]),
                     simpleUnexplode: .custom("red,green,blue", value: ["": ["red", "green", "blue"]]),
                     formDataExplode: "list=red&list=green&list=blue",
-                    formDataUnexplode: "list=red,green,blue"
+                    formDataUnexplode: "list=red,green,blue",
+                    deepObjectExplode: "list=red&list=green&list=blue"
```

Comment:
Shouldn't this have some square brackets? Or be unsupported, as an array?

Reply:
That's good - then this test should reflect that. Either by catching an expected error or expecting an empty parsed dictionary.

Reply:
Completed. I have added support for testing the errors thrown as well. However, I'm unsure about whether it's correct that I made `ParsingError` and `SerializationError` public. I would appreciate your feedback on this.